src/ui-test/extensionUITest.ts

```diff
@@ -0,0 +1,57 @@
+import { expect } from 'chai';
+import {
+  ActivityBar,
+  SideBarView,
+  ViewControl,
+  ExtensionsViewSection,
+  InputBox,
+} from 'vscode-extension-tester';
+
+/**
+ * @author Ondrej Dockal <odockal@redhat.com>
+ */
+export function extensionUIAssetsTest(): void {
+  describe('Verify base assets are available after installation', () => {
+    let view: ViewControl;
+    let sideBar: SideBarView;
+    let quickBox: InputBox;
+
+    before(async function () {
```

Comment:
Nit: why isn't it an arrow function? This is inconsistent with the outer lambda. Does `this` need to be bound to a different context?

Reply:
It is mocha thing, arrow function wont let you get into mocha context to set timeout for the test: https://mochajs.org/#arrow-functions